File: Api/Models/ServerEvents/ServerNotifiesClientsInRoomSomeoneHasJoinedRoom.cs
using Api.Models.QueryModels;
using lib;

namespace Api.Models.ServerEvents;

public class ServerNotifiesClientsInRoomSomeoneHasJoinedRoom : BaseDto
{
    public int roomId { get; set; }

    public string? message { get; set; }
    public string userEmail { get; set; }
}

File: frontend/src/models/serverNotifiesClientsInRoomSomeoneHasJoinedRoom.ts
import {BaseDto} from "./baseDto";
import {EndUser} from "./entities";

export class ServerNotifiesClientsInRoomSomeoneHasJoinedRoom extends BaseDto<ServerNotifiesClientsInRoomSomeoneHasJoinedRoom> {
  userEmail?: EndUser
  roomId?: number;
  message?: string;
}
